contentcuration/contentcuration/view/settings_views.py

```diff
@@ -37,7 +37,7 @@ class ProfileView(FormView):
     def get_context_data(self, **kwargs):
         context = super(ProfileView, self).get_context_data(**kwargs)
         channel_list = Channel.objects.filter( Q(deleted=False, editors= self.request.user)).values("id", "name")
-        context.update({'channel_list': channel_list, "page": "profile", 'channel_name': False})
+        context.update({'channel_list': channel_list, "page": "profile", 'channel_name': False, "success":False})
```

Comment:
Right now, CC will throw an error that this key is not present, making it difficult to find logs on the error log. This will fix this so that the error log will be a little bit easier to look through